### PR TITLE
Passes User creation test

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * Photo and rating weren't defined correctly in `model.py` file
 
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing. 
- [x] Yes, and all are passing.
 
### Message/Questions for reviewer:

In order to get all tests to pass, one must have the server running in another tab/window

 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: main_app/models.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, \
+                                        PermissionsMixin
+class Park(models.Model):
+    """Makes park instance for favorited park"""
+    name = models.CharField(max_length=300)
+    formatted_address = models.CharField(max_length=300)
+    opening_hours = models.CharField(max_length=1000, default='')
+    photo = models.CharField(max_length=1000, default='')
+    rating = models.CharField(max_length=100, default='0')
```
Comment (TravisGM92) [inline main_app/models.py:10]: Line 9 and 10 had `models.CharField` which is why they weren't working. They needed arguments `()`

File: main_app/migrations/0002_park.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        ('main_app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Park',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=300)),
+                ('address', models.CharField(max_length=300)),
```
Comment (judithpillado) [inline main_app/migrations/0002_park.py:18]: will it also include photo, rating, opening hours, etc?

File: main_app/models.py (hunk 1 above)
Comment (judithpillado) [reply main_app/models.py:10]: thank you for clarifying

**Review by judithpillado — APPROVED** [review]

File: main_app/models.py (hunk 1 above)
Comment (robertorodriguez12) [reply main_app/models.py:10]: Thanks Travis! I really had some struggle with that part of it.

**Pull request merged**